nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-                    },
-                    {
-                      "year": 2002,
-                      "value": 15
-                    }
-                  ]
-                },
-                {
-                  "label": "Fine Sea Salt",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 20
-                    },
-                    {
-                      "year": 2001,
-                      "value": 25
-                    },
-                    {
-                      "year": 2002,
-                      "value": 17
-                    }
-                  ]
-                },
-                {
-                  "label": "Fine Soil",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 0.5
-                    },
-                    {
-                      "year": 2001,
-                      "value": 0.75
-                    },
-                    {
-                      "year": 2002,
-                      "value": 0.20
-                    }
-                  ]
-                }
-              ]
-            },
-            {
-              "name": "Haziest Days by Year",
-              "yAxisLabel": "Light Extinction, 1/Mm",
-              "variableLines": [
-                {
-                  "label": "Coarse Mass",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 10
-                    },
-                    {
-                      "year": 2001,
-                      "value": 11
-                    },
-                    {
-                      "year": 2002,
-                      "value": 15
-                    }
-                  ]
-                },
-                {
-                  "label": "Fine Sea Salt",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 20
-                    },
-                    {
-                      "year": 2001,
-                      "value": 25
-                    },
-                    {
-                      "year": 2002,
-                      "value": 17
-                    }
-                  ]
-                },
-                {
-                  "label": "Fine Soil",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 0.5
-                    },
-                    {
-                      "year": 2001,
-                      "value": 0.75
-                    },
-                    {
-                      "year": 2002,
-                      "value": 0.20
-                    }
-                  ]
-                }
-              ]
-            }
-          ]
-        }
-      ]
-    },
-    "ozone": {
-    },
-    "sulfurDeposition": {
-    }
-  ]
-}
+  {
```

Comment:
disregard green area below. I'm hacking GitHub to allow for line-comments and this is a side-effect :)

Reply:
Nice.